src/app_service/service/contacts/service.nim

```diff
@@ -604,8 +604,9 @@ QtObject:
         return self.receivedIdentityRequests[publicKey]
 
       let response = status_contacts.getVerificationRequestFrom(publicKey)
-      result = response.result.toVerificationRequest()
-      self.receivedIdentityRequests[publicKey] = result
+      if not response.result.isNil and response.result.kind == JObject:
+        result = response.result.toVerificationRequest()
+        self.receivedIdentityRequests[publicKey] = result
```

Comment:
This is a minor comment but I find "getVerificationRequestFrom" a bit misleading - it gets verReq from status_go but also sets it in receivedIdentityRequests, unlike "getVerificationRequestSentTo" which only gets.